### PR TITLE
Add missing diagnostics

### PR DESCRIPTION
**Pull request by jasonblais**
- https://github.com/mattermost/mattermost-server/pull/8862
- https://github.com/mattermost/mattermost-server/pull/8699
- https://github.com/mattermost/mattermost-server/pull/8756
- https://github.com/mattermost/mattermost-server/pull/8800
- https://github.com/mattermost/mattermost-server/pull/8881

### CODE REVIEW

File: app/diagnostics.go [hunk 1]
```diff
@@ -517,6 +521,7 @@ func (a *App) trackConfig() {
 
 	a.SendDiagnostic(TRACK_CONFIG_DISPLAY, map[string]interface{}{
 		"experimental_timezone": *cfg.DisplaySettings.ExperimentalTimezone,
+		"custom_url_schemes":    isDefault(*cfg.DisplaySettings.CustomUrlSchemes, ""),
```
Comment (jasonblais) [inline app/diagnostics.go]: @hmhealey Not sure if this is correct

File: app/diagnostics.go [hunk 2]
```diff
@@ -401,6 +404,7 @@ func (a *App) trackConfig() {
 		"isdefault_nickname_attribute":        isDefault(*cfg.LdapSettings.NicknameAttribute, model.LDAP_SETTINGS_DEFAULT_NICKNAME_ATTRIBUTE),
 		"isdefault_id_attribute":              isDefault(*cfg.LdapSettings.IdAttribute, model.LDAP_SETTINGS_DEFAULT_ID_ATTRIBUTE),
 		"isdefault_position_attribute":        isDefault(*cfg.LdapSettings.PositionAttribute, model.LDAP_SETTINGS_DEFAULT_POSITION_ATTRIBUTE),
+		"isdefault_login_id_attribute":        isDefault(*cfg.LdapSettings.LoginIdAttribute, model.""),
```
Comment (hmhealey) [inline app/diagnostics.go]: The default value should be `""`, not `model.""`

File: app/diagnostics.go (hunk 1 above)
Comment (hmhealey) [reply app/diagnostics.go]: I don't think `isDefault` will work here, but we could use `len(*cfg.DisplaySettings.CustomUrlSchemes) != 0` to check if they've changed it from the default empty array

**Comment by jasonblais** [comment]
@hmhealey Updated

**Pull request merged**